fitnessllm_dataplatform/infrastructure/RedisConnect.py

```
@@ -0,0 +1,105 @@
+import redis
+
+from fitnessllm_dataplatform.utils.cloud_utils import get_secret
+from fitnessllm_dataplatform.utils.logging_utils import logger
+
+
+class RedisConnect:
+    """Infrastructure Redis Connect."""
+
+    def __init__(self):
+        """Init function."""
+        pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)